Formula/cloudflare-quiche.rb

```diff
@@ -21,10 +21,22 @@ class CloudflareQuiche < Formula
   depends_on "rust" => :build
 
   def install
+    system "cargo", "build", "--features", "ffi"
+    lib.install "target/debug/libquiche.dylib"
```

Comment:
Since Homebrew supports both macOS and Linux, a common idiom is "`.so` or `.dylib` depending on the platform". For this, you can use `shared_library("libquiche")`, which expands to `libquiche.dylib` on macOS and `libquiche.so` on Linux.

You can interpolate this in the string, like `lib.install "path/to/#{shared_library("libfoo")}"`, see example usage [in the `resvg` formula](https://github.com/Homebrew/homebrew-core/blob/ed1771b21ae1d86d9d0ee0e8587e1879ce2e39f7/Formula/resvg.rb#L27).